pkg/compactor/block_upload_test.go

```diff
@@ -702,6 +734,7 @@ func TestMultitenantCompactor_UploadBlockFile(t *testing.T) {
 		blockID                string
 		path                   string
 		body                   string
+		enableBlockUpload      bool
```

Comment:
same comment about `disableBlockUpload`, it will make PR diff smaller thanks to Go's default value being false.

Reply:
Done.